data/campaigns/The_Deceivers_Gambit/scenarios/14_Long_Live_the_Queen.cfg

```diff
@@ -970,19 +970,21 @@ You like magic so much? Once I drink from Jevyan’s Cup, I’ll have enough mag
         #############################
         # DIALOGUE
         #############################
+#define MAGIC_ELDRED_MESSAGE STRING
+"<span color='#CC0000' size='small' font_weight='bold'>" + {STRING} + "</span>"#enddef
```

Comment:
Looks like the indent check is confused by the `#enddef` not being on a new line...?